modules/heat_transfer/doc/content/source/kernels/ADHeatConductionTimeDerivative.md

```diff
@@ -2,22 +2,19 @@
 
 ## Description
 
-The `ADHeatConductionTimeDerivative` kernel implements a time derivative for the domain $\Omega$ given by
-
-\begin{equation}
-\underbrace{\rho c_p \frac{\partial u}{\partial t}}_{\textrm{ADHeatConductionTimeDerivative}} +
-\sum_{i=1}^n \beta_i = 0 \in \Omega.
-\end{equation}
-where $\rho$ is the material density, $c_p$ is the specific heat, and the second term on the left hand side corresponds to the strong forms of
-other kernels. The corresponding `ADHeatConductionTimeDerivative` weak form using inner-product notation is
-
-\begin{equation}
-R_i(u_h) = (\psi_i, \rho c_p\frac{\partial u_h}{\partial t}) \quad \forall \psi_i,
-\end{equation}
-where $u_h$ is the approximate solution and $\psi_i$ is a finite element test function.
-
-The Jacobian is given by automatic differentiation, and should be perfect as long as $c_p$ and $\rho$
-are provided using `ADMaterial` derived objects.
+`ADHeatConductionTimeDerivative` is the implementation of [HeatConductionTimeDerivative](/HeatConductionTimeDerivative.md) within the framework of [!ac](AD). Please see the [HeatConductionTimeDerivative](/HeatConductionTimeDerivative.md) documentation for more information.
+
+For this object, the diffusion coefficient can either be an `ADMaterial` or traditional `Material`, though the Jacobian will be perfect if using an `ADMaterial`.
```

Comment:
The previous documentation is not very good. This object operates with density and specific heat capacity,  not diffusion/conductivity

Reply:
Also need to update the language about `Material` vs `ADMaterial`